src/test/java/org/folio/inventory/support/http/client/OkapiHttpClientTests.java

```diff
@@ -41,7 +42,8 @@
   private static VertxAssistant vertxAssistant;
 
   @Rule
-  public WireMockRule fakeWebServer = new WireMockRule();
+  public WireMockRule fakeWebServer = new WireMockRule(wireMockConfig()
+    .dynamicPort());
```

Comment:
Is this because there were port conflicts whilst running these tests?

Reply:
Yes, as noted in the PR comments.

Reply:
Apologies for not checking the notes